### ISSUE TITLE
Open file at specific row:column (necessary for SyncTeX)

### ISSUE THREAD

**Issue by joachimneu** [howl-editor/howl#478]
TL;DR: Is there a way to enable SyncTeX with Howl? I could not find any, neither in the documentation nor somewhere else online, nor was I able to set it up myself. SyncTeX requires some IPC mechanism to trigger the following action in an already running Howl instance: "navigate to a certain row and column of a certain file". (Info about SyncTeX: https://tex.stackexchange.com/a/118491)

If you agree that there is currently no way to do this in Howl, I see two ways how this could be achieved:

1. Rather straight-forward for this specific use case: Add optional parameters ":row:column" to the path of a file to be opened, and if Howl sees that, it opens the file and automatically navigates the cursor to the respective location.

2. The more general approach, perhaps improves Howl interoperability/scriptability/remote-controllability for other use cases as well: Expose the Howl command line interface through some IPC mechanism, for instance DBus, or "howl --reuse --cmd 'set font_size 12'".

**Comment by nilnor** [comment]
I implemented the option since this is something that has been on the back of the list for quite a while. Works both when running Howl locally or when remotely using `--reuse`.

**Comment by joachimneu** [comment]
Awesome, thank you! :)
